neard/src/cli.rs

```diff
@@ -431,6 +431,10 @@ fn init_logging(verbose: Option<&str>) {
         }
     }
 
+    if cfg!(feature = "sandbox") {
+        env_filter = env_filter.add_directive("sandbox=debug".parse().unwrap());
```

Comment:
Could you add a comment describing why this is being done? Something along the lines of

> The `env.sandbox_debug_log` VMLogic function implements output via this `sandbox` logging target.